src/stdlib/src/lib.rs

```diff
@@ -712,9 +740,8 @@ pub extern "C" fn wctomb(s: *mut c_char, wchar: wchar_t) -> c_int {
     unimplemented!();
 }
 
-/*
-#[no_mangle]
-pub extern "C" fn func(args) -> c_int {
-    unimplemented!();
-}
-*/
+// #[no_mangle]
+// pub extern "C" fn func(args) -> c_int {
```

Comment:
This can be removed

Reply:
Why was this even there in the first place lol